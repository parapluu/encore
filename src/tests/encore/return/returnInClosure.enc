fun id[t](x : t) : t
  return x
end

active class Main
  def main() : unit
    val f = fun (x : int) : int
              if x > 0 then
                println("Inside the function")
                return x + 1
                println("After the return")
                0
              else
                x
              end
            end
    println(f(41))
  end
end