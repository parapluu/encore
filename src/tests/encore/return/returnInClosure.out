Inside the function
42
